Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -150,7 +154,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("int")
                         .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
 
-                    b.Property<int?>("RoleId")
```

Comment:
Might cause an issue if we change a nullable to not nullable. But again I'm not familiar with migrations and if this is migrations doing its magic it should be good.

Reply:
To be honest i'm not sure why this is changed. @Brend-Smits do you know why? Should I revert it?

Reply:
No, keep it. I mentioned this earlier but I think we forgot about a migration earlier. Good question!